models/pardot__campaigns.sql

```diff
@@ -1,13 +1,18 @@
-{% set statuses = dbt_utils.get_column_values(table=ref('int__opportunity_tmp'), column='opportunity_status') %} 
+{% if execute and flags.WHICH in ('run', 'build') %}
+    {% set statuses = dbt_utils.get_column_values(table=ref('int__opportunity_tmp'), column='opportunity_status') %} 
+{% else %}
+    {% set statuses = [] %}
+{% endif %}
```

Comment:
Suggestion for conciseness.

```suggestion
 {% set statuses = dbt_utils.get_column_values(table=ref('int__opportunity_tmp'), column='opportunity_status') if execute and flags.WHICH in ('run', 'build') else [] %} 
```